code/services/QueuedJobService.php

```diff
@@ -325,6 +325,9 @@ public function checkJobHealth($queue = null) {
 	 */
 	public function checkdefaultJobs($queue = null) {
 		$queue = $queue ?: QueuedJob::QUEUED;
+
+		$this->defaultJobs = !empty($this->defaultJobs) ? $this->defaultJobs : Config::inst()->get(__CLASS__, 'defaultJobs');
```

Comment:
The point is that it should never be calling `Config::inst()->get(__CLASS__, 'defaultJobs');` - defaultJobs is declared as a member variable (which should be injector set) rather than a private static (Config:: managed).